src/wp-admin/includes/ajax-actions.php

```diff
@@ -3970,23 +3970,43 @@ function wp_ajax_crop_image() {
 			/** This filter is documented in wp-admin/includes/class-custom-image-header.php */
 			$cropped = apply_filters( 'wp_create_file_in_uploads', $cropped, $attachment_id ); // For replication.
 
-			$parent_url = wp_get_attachment_url( $attachment_id );
-			$url        = str_replace( wp_basename( $parent_url ), wp_basename( $cropped ), $parent_url );
+			$parent_url      = wp_get_attachment_url( $attachment_id );
+			$parent_basename = wp_basename( $parent_url );
+			$url             = str_replace( $parent_basename, wp_basename( $cropped ), $parent_url );
 
 			$size       = wp_getimagesize( $cropped );
 			$image_type = ( $size ) ? $size['mime'] : 'image/jpeg';
 
+			// Get the original image's post to pre-populate the cropped image.
+			$original_attachment      = get_post( $attachment_id );
+			$use_original_title       = (
+				'' !== trim( $original_attachment->post_title ) &&
+				// Check if the original image's title was edited.
+				pathinfo( $parent_basename, PATHINFO_FILENAME ) !== $original_attachment->post_title
```

Comment:
@anton-vlasenko I added this check to only use the original image's title _if it was edited_. When there's no title in the image's metadata, by default the image's filename is set as the `post_title`. This check is making sure the `post_title` is not the same as that default behavior. If no, then it will use the original image's title.

Reply:
The tests will need updating though.

Reply:
Thank you!
I'm working on updating the tests.